### ISSUE TITLE
Custom Solutions guide links to Getting Started From Scratch, and that guide links to kubectl install which already seems to assume I have kube-up.sh

### ISSUE THREAD

**Issue by JonasT** [kubernetes/kubernetes#45102]
I found the following isuse with the documentation:

The Custom Solutions ( https://kubernetes.io/docs/setup/pick-right-solution/#custom-solutions )guide links to Getting Started From Scratch ( https://kubernetes.io/docs/getting-started-guides/scratch/ ), and that guide links to kubectl install which already seems to assume I have kube-up.sh ( https://kubernetes.io/docs/tasks/kubectl/install/ ).

In the section "Configuring kubectl" the tool kube-up.sh is just casually mentioned without any links to relevant documentation, and "Making sure it's ready" seems to assume for `kubectl cluster-info` I already used it to create a cluster. I can click the "getting started guides" link in that paragraph to get back to https://kubernetes.io/docs/getting-started-guides/ but that is where I just came from!

TL;DR: Minikube cluster docs are easy to find from kubectl install guide, but if I want to create a proper cluster directly on the host there is no obvious link from the kubectl install guide where I should go to read up on it.

**Comment by JonasT** [comment]
On second examination, I suggest this paragraph should be expanded:

> Learning
>
>    You should be familiar with using Kubernetes already. We suggest you set up a temporary cluster by following one of the other Getting Started Guides. This will help you become familiar with the CLI (kubectl) and concepts (pods, services, etc.) first.

It doesn't actually link a guide that should be started with first (which explains on how to do a basic kubernetes install). This is why I ended up on kubectl instead, which isn't really a good place to start if you have never set up a cluster and you want to have one.

**Comment by kargakis** [comment]
@kubernetes/sig-docs-maintainers 
/sig docs

**Comment by JonasT** [comment]
For what it's worth, this seems to be fixed anyway. It clearly refers to guides right below, where the kubeadm guide is on the very top. I think that should make it much easier for people to find this than the way it was written when I came to this page.
